java-checks/src/test/files/checks/security/EncryptionAlgorithmCheck.java

```diff
@@ -0,0 +1,67 @@
+import javax.crypto.Cipher;
+import java.security.NoSuchAlgorithmException;
+import javax.crypto.NoSuchPaddingException;
+import java.security.NoSuchProviderException;
+
+abstract class A {
+
+  public void foo(java.util.Properties props) {
+    /*
+    should complain:
+    - everytime ECB mode is used whatever the encryption algorithm
+       - By default without specifying operation mode ECB is chosen
+    - when CBC mode is used with PKCS5Padding or PKCS7Padding
+    - when RSA is used without OAEPWithSHA-1AndMGF1Padding or OAEPWITHSHA-256ANDMGF1PADDING padding scheme
+    */
+
+    try
```

Comment:
usually, in java, brackets are on the same line as the keyword. Can you fix this?